src/tools/annotation/FreehandRoiTool.js

```diff
@@ -145,6 +147,14 @@ export default class FreehandRoiTool extends BaseAnnotationTool {
     this.onContourRightClicked = onContourRightClicked;
   }
 
+  setOnContourCopyHotKeyPressed(onContourCopyHotKeyPressed) {
```

Comment:
Although this is in the FreehandRoi tool, you should name these callbacks more generally, since the concept of "copy and paste" is external to the keys themselves. Something like `SetCtrlCKeyPressed` or something

Reply:
c50035d266e2c1e89b9552777e1d9546668a5879